subsys/bluetooth/audio/vcp_vol_rend.c

```diff
@@ -192,9 +192,11 @@ static ssize_t write_vcs_control(struct bt_conn *conn,
 	if (volume_change && !vol_rend.flags) {
 		vol_rend.flags = 1;
 
+#if defined(CONFIG_BT_VCP_VOL_REND_VOL_FLAGS_NOTIFIABLE)
 		bt_gatt_notify_uuid(NULL, BT_UUID_VCS_FLAGS,
 				    vol_rend.service_p->attrs,
 				    &vol_rend.flags, sizeof(vol_rend.flags));
+#endif /* CONFIG_BT_VCP_VOL_REND_VOL_FLAGS_NOTIFIABLE */
```

Comment:
Since this is not using any guarded values, suggest to use `IS_ENABLED` instead

Reply:
What do you mean by guarded values?
I just read the FAQ on IS_ENABLED and I understand the benefits - but when should we not use it?
Or is it beneficial to just replace all `defined` with `IS_ENABLED`?

Reply:
Doing 
```c
	if (IS_ENABELD(CONFIG_FOO)) {
		/* something */
	}
```
is preferred over 
```c
#if defined(CONFIG_FOO)
	/* something */
#endif
```

But if `/* something */` contains access to static functions that are guarded by `CONFIG_FOO` (i.e. not compiled) or if it contains access to struct members that are also guarded by `CONFIG_FOO` then we cannot use `IS_ENABLED`. 

In this case here, everything in 
```c
		bt_gatt_notify_uuid(NULL, BT_UUID_VCS_FLAGS,
				    vol_rend.service_p->attrs,
				    &vol_rend.flags, sizeof(vol_rend.flags));
```
is always compiled, so we can use IS_ENABLED